Quickstart/00-Starter-Seed/Controllers/ApiController.cs

```diff
@@ -8,7 +8,7 @@ public class ApiController : ControllerBase
     {
         [HttpGet]
         [Route("public")]
-        public IActionResult Public()
+        public ActionResult Public()
```

Comment:
Why was it necessary to change the IActionResult interface types to the concrete type?

Reply:
It wasn't really necessary. It just provides a handful of advantages over `IActionResult`, especially if you're generating swagger docs. In ASP.NET Core 2.2+ it's generally recommended over the interface, so I'm just following best practice.

Reply:
I couldn't find anything about why to use it over IActionResult - do you have a link?

Reply:
https://joonasw.net/view/aspnet-core-2-1-actionresult-of-t
https://docs.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-3.1#actionresultt-type

It's a minor improvement, but an improvement nonetheless.

Reply:
Those examples are specifically talking all about the benefits of using `ActionResult<T>` type and the extra inference they can do knowing the `T` type for Swagger etc.

I couldn't see any benefit of just using the non-generic `ActionResult`. Given that all these methods actually return anonymous objects there isn't a T to use for the generic version.

Reply:
That's a valid point. I can revert that change if you'd like.

Reply:
Yeah lets go back to the IActionResult for now - the other changes look great, thanks!